docs/reference/configuration/data_types.rst

```diff
@@ -72,167 +61,351 @@ This section only applies if a fixed seed is used:
 
-TRUNCATED_NORMAL
-,,,,,,,,,,,,,,,,,
+LOGNORMAL: Log Normal Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This *TRUNCATED_NORMAL* distribution works as follows:
+The ``LOGNORMAL`` keyword is used to assign a log normal prior to a variable. A variable is considered log normally distributed if the logarithm of that variable follows a normal distribution. 
+In mathematical terms, if :math:`X` is normally distributed, then :math:`Y = e^X` is log normally distributed.
 
-   1. Draw random variable :math:`X \sim N(\mu,\sigma)`
-   2. Clamp X to the interval [min, max]
+Log normal priors are especially suitable for modeling positive quantities that exhibit a heavy tail, indicating a tendency for the quantity to occasionally take large values.
 
-This is **not** a proper normal distribution; hence the
-clamping to `[min,max]` should be an exceptional event. To configure
-this distribution for a situation with mean 1, standard deviation 0.25
-and hard limits 0 and 10:
+Syntax
+~~~~~~
+::
+
+  VAR LOGNORMAL <log_mean> <log_standard_deviation>
+
+Parameters
+~~~~~~~~~~
+- **<log_mean>**: Represents the mean of the logarithm of the variable.
+- **<log_standard_deviation>**: Denotes the standard deviation of the logarithm of the variable.
 
+Example
+~~~~~~~
+For a variable with its logarithm having a mean of 0 and a standard deviation of 1:
 ::
 
-   VAR3  TRUNCATED_NORMAL  1  0.25   0  10
+   VAR LOGNORMAL 0 1
 
+.. image:: fig/lognormal.png
 
-UNIFORM
-,,,,,,,
+TRUNCATED_NORMAL: Truncated Normal Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A stochastic variable is uniformly distributed if has a constant
-probability density on a closed interval. Thus, the uniform
-distribution is completely characterized by it's minimum and maximum
-value. To assign a uniform distribution to a variable, use the keyword
-UNIFORM, which takes a minimum and a maximum value for a the
-variable. Here is an example, which assigns a uniform distribution
-between 0 and 1 to a variable ``VAR4``:
+The ``TRUNCATED_NORMAL`` keyword is utilized to assign a truncated normal distribution to a variable.
+This disribution works as follows:
 
+   1. Draw a random variable :math:`X \sim N(\mu,\sigma)`.
+   2. Clamp :math:`X` to the interval [min, max].
+
+It's important to note that this is **not** a proper normal distribution due to the constraints applied.
+Thus, the act of clamping values to the `[min,max]` interval should be an exceptional event. 
+
+Syntax
+~~~~~~
 ::
 
-   VAR4 UNIFORM 0 1
+  VAR TRUNCATED_NORMAL <mean> <standard_deviation> <min> <max>
+
+Parameters
+~~~~~~~~~~
+- **<mean>**: Specifies the mean of the normal distribution prior to truncation.
+- **<standard_deviation>**: Indicates the standard deviation of the distribution before truncation.
+- **<min>**: Determines the lower truncation limit.
+- **<max>**: Establishes the upper truncation limit.
+
+Example
```

Comment:
Emphasize that this is an example showing how this should not be used? "Bad example", followed by a "Good example"?

Reply:
What's a good example?

Reply:
Where the clamping is an "exceptional event", so that there is no visible spikes at the truncation limit.

Reply:
I have added an example of what I think is reasonable usage of this distribution.